topics/climate/tutorials/ocean_qcv_analysis/tutorial.md

```diff
@@ -0,0 +1,346 @@
+> > 1. Open your favorite browser (Chrome, Safari or Firefox as your browser, not Internet Explorer!)
+> > 2. Browse to your [Galaxy instance](https://earth-system.usegalaxy.eu/)
+> > 3. On the top panel go to **Login or Register**
+> >
+> {: .hands_on}
+>
+> The Galaxy homepage is divided into three panels:
+> * Tools on the left
+> * Viewing panel in the middle
+> * History of analysis and files on the right
+>
+> ![Screenshot of the Galaxy interface, the tools panel is on the left, the main panel is in the center, and the history is on the right.]({% link shared/images/galaxy_interface.png %} "Galaxy interface explanation")
+>
+> The first time you use Galaxy, there will be no files in your history panel.
+{: .details}
+
+# Manage your data 
+
+> <hands-on-title>Get data</hands-on-title>
+> Upload the argo data files (meta, core and BGC) of the float 4903881 from the S3 service. 
+> - 2a. Click on “Upload” on the vertical panel on the left. A pop-up is launched
+> - 2b. Click on “Choose from repository” at the bottom of the popup
+>
+> ![Image of how to uploade remote data files](../../images/bgc_calib/bgc_2a.png)
+>
+> - 2c. Search “argo” in the top search bar. “Argo marine floats data and metadata from Global Data Assembly Centre (Argo GDAC)” appears in the label column just below
+> - 2d. Select it then “pub/dac/coriolis/4903881” then tick “4903881_meta.nc” + “profiles/”
+> - 2e. Click on “Select” on the bottom right of the pop-up. the list of the file to download is displayed
+> - 2f. Remove the Synthetic files (not useful for qualification & calibration actions)  by clicking on the corresponding garbage can on the right
+> - 2g. Click on “start” on the bottom of the pop-up. Once the download is complete (green color), files are stored in your history on the right.
+>
+> ![Image of the loading of the data files](../../images/bgc_calib/bgc_2f.png)
+{: .hands_on}
+
+> <hands-on-title>Organize your data</hands-on-title>
+> Create a dataset list/collection of the argo float 4903881 by clicking on
+> - 3a. “Select item” 
+> - 3b.  “Select all” (appears just after 3a and at the same level on the right)
+> - 3c.  “All ## selected”
+> - 3d.  “Auto build list” >> a pop-up is launched
+> - 3e.  Enter the name of the data collection : 4903881
+> - 3f.   turn off “Remove the file extension”
+> - 3g.  Click on “Build” on the bottom right of the pop-up
+>
+> ![Image of how to build a collection from multiple netcdf files](../../images/bgc_calib/bgc_3a.png)
+> 
+>
+> Once the collection is ready, everything is green in the history panel. It is possible to limit the history to useful files or collections by clicking on the eyes above the files list in the history.
+{: .hands_on}
+
+
+
+{% include _includes/cyoa-choices.html option1="Tools" option2="Workflow" default="tools" disambiguation="tutorialstyle"
+       text="Do you want to run the workflow or to discover the tools one by one ?" %}
+
+<div class="Workflow" markdown="1">

```

Comment:
I'm just guess, but you specified option1="Tools" but the first one there is "Workflows", can you change the order? What should be the default?